zer0-os/zOS
# Add sentry

### Testing Steps
- Add the Sentry DSN (https://sentry.io/settings/factory-5k/projects/zos/keys/) to your `.env` file
- Throw an error from a component in your local app
- Verify the exception is recorded in Sentry (https://sentry.io/organizations/factory-5k/projects/zos/?issuesType=all&project=6751654)

### Screen Capture
![Screen-Shot-2022-09-27-at-2-49-20-PM](https://user-images.githubusercontent.com/31045/192613397-c4db0120-a368-4404-b75b-ff11f5606254.png)
![Screen-Shot-2022-09-27-at-2-49-11-PM](https://user-images.githubusercontent.com/31045/192613403-e8db0dee-a1a8-436f-b83d-16bb2d758923.png)
